db/errorHandlingControllers.js

```diff
@@ -0,0 +1,7 @@
+exports.handle404Status = (error, request, response, next) => {
```

Comment:
I suspect only your first error handler ever gets invoked currently and is used for your invalid endpoint test. Remove as part of our discussion involving invalid endpoints.